stories/object-identifier/object-identifier.stories.js

```diff
@@ -0,0 +1,173 @@
+import '../../dist/object-identifier.css';
+import '../../dist/link.css';
+import '../../dist/table.css';
+
+export default {
+    title: 'Components/Object Identifier',
```

Comment:
Can we create a folder for all of these "Object" components?

Reply:
Hi @jbadan 
Do you mean move all folders (`object-identifier`, `object-marker`, `object-number`, `object-status`) into one `objects` folder?

Reply:
We can table it for now - I think this is more of a question for @jessmetivier